crates/e2e/tests/e2e/colocation_buffers.rs

```diff
@@ -0,0 +1,143 @@
+use {
+    e2e::{setup::*, tx},
+    ethcontract::prelude::U256,
+    model::{
+        order::{OrderCreation, OrderKind},
+        signature::EcdsaSigningScheme,
+    },
+    secp256k1::SecretKey,
+    shared::ethrpc::Web3,
+    web3::signing::SecretKeyRef,
+};
+
+#[tokio::test]
+#[ignore]
+async fn local_node_buffers() {
+    run_test(onchain_settlement_without_liquidity).await;
+}
+
+async fn onchain_settlement_without_liquidity(web3: Web3) {
+    let mut onchain = OnchainComponents::deploy(web3).await;
+
+    let [solver] = onchain.make_solvers(to_wei(1)).await;
+    let [trader] = onchain.make_accounts(to_wei(1)).await;
+    let [token_a, token_b] = onchain
+        .deploy_tokens_with_weth_uni_v2_pools(to_wei(1_000), to_wei(1_000))
+        .await;
+
+    // Fund trader, settlement accounts, and pool creation
+    token_a.mint(trader.address(), to_wei(100)).await;
+    token_b
+        .mint(onchain.contracts().gp_settlement.address(), to_wei(5))
+        .await;
+    token_a.mint(solver.address(), to_wei(1000)).await;
```

Comment:
IIRC we now have a system for tests which should not require all these manual `tx!` calls to make e2e more readable.